src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -36,7 +36,12 @@ public class AzureAppConfigurationOptions
 
         // The following set is sorted in descending order.
         // Since multiple prefixes could start with the same characters, we need to trim the longest prefix first.
-        private SortedSet<string> _keyPrefixes = new SortedSet<string>(Comparer<string>.Create((k1, k2) => -string.Compare(k1, k2, StringComparison.OrdinalIgnoreCase)));
+        private SortedSet<string> _keyPrefixes = new SortedSet<string>(Comparer<string>.Create((k1, k2) => -string.Compare(k1, k2, StringComparison.OrdinalIgnoreCase)));
+
+        /// <summary>
+        /// Flag to indicate whether discover the replicas automatically.
+        /// </summary>
+        public bool AutoDiscoverReplica { get; set; } = true;
```

Comment:
```suggestion
        public bool EnableReplicaDiscovery { get; set; } = true;
```

Reply:
See [this comment](https://github.com/microsoft/FeatureManagement-Dotnet/pull/259#discussion_r1341882283)

My suggestion should be `ReplicaDiscoveryEnabled` in order to avoid a verb as a property name.